File: week3/Choi/Week3/HouseBlend.cs
namespace Week3
{
    public class HouseBlend : Beverage
    {
        public HouseBlend() : base()
        {
            m_Description = "하우스 블랜드";
        }

        public override double Cost()
        {
            return 0.89f;
        }
    }
}

File: week3/rlawjddbs/decorator/beverage/HouseBlend.java
package week3.decorator.beverage;

public class HouseBlend extends Beverage {

	public HouseBlend() {
		description = "하우스 블렌드 커피";
	} // basic constructor

	@Override
	public double cost() {
		return .89;
	} // cost

} // class
